maven-model-transform/src/test/java/org/apache/maven/model/transform/ConsumerPomXMLFilterTest.java

```diff
@@ -82,10 +69,9 @@ protected AbstractSAXFilter getFilter( Consumer<LexicalHandler> lexicalHandlerCo
 
         };
 
-        RawToConsumerPomXMLFilter filter =
-            new RawToConsumerPomXMLFilterFactory( buildPomXMLFilterFactory ).get( Paths.get( "pom.xml" ) );
-        filter.setFeature( "http://xml.org/sax/features/namespaces", true );
-        return filter;
+        parser = new RawToConsumerPomXMLFilterFactory( buildPomXMLFilterFactory )
+                        .get( parser, Paths.get( "pom.xml" ) );
+        return parser;
```

Comment:
Same comment here, but in this case you can directly return the statement: `return new RawToConsumerPomXMLFilterFactory(...`

Reply:
I really don't get the problem, but once again, I've added an intermediary variable.
Some people also prefer not returning instances created directly to ease debugging.